Makefile

```diff
@@ -96,8 +96,8 @@ assets:
 database:
 	$(call docker-exec-backend,console doctrine:database:drop --no-interaction --force)
 	$(call docker-exec-backend,console doctrine:database:create --no-interaction)
-	$(call docker-exec-backend,console doctrine:schema:update --no-interaction --force)
-	$(call docker-exec-backend,console doctrine:fixtures:load --no-interaction --append)
+	$(call docker-exec-backend,console doctrine:migrations:migrate --no-interaction)
```

Comment:
shall we do this change? IMO if the messenger issue is fixed, maybe this is not needed anymore?

Reply:
You're right, maybe I can change this line in another PR if its necessary :)